config/default.yml

```diff
@@ -196,6 +196,11 @@ Performance/StartWith:
   VersionAdded: '0.36'
   VersionChanged: '0.44'
 
+Performance/StringInclude:
+  Description: 'Use `String#include?` instead of a regex match with literal-only pattern.'
+  Enabled: true
+  VersionAdded: '1.6'
```

Comment:
Can you update from `'1.6'` to to `'1.7'` because patch version does not add a new cop?